framework/console/controllers/HelpController.php

```diff
@@ -532,7 +531,7 @@ protected function formatOptionHelp($name, $required, $type, $defaultValue, $com
     protected function formatOptionAliases($controller, $option)
     {
         foreach ($controller->optionAliases() as $name => $value) {
-            if (Inflector::camel2id($value, '-', true) === $option) {
+            if ($this->camel2id($value) === $option) {
```

Comment:
Any explanation why this was changed? Original issue was only about action names.